_posts/en/newsletters/2020-02-19-newsletter.md

```diff
@@ -0,0 +1,296 @@
+       k-of-n threshold signing, and adaptor signatures (scriptless
+       scripts) that look like single-sig spends onchain.
+
+       Yet as more people turn to multisig and advanced contracts,
+       there's an increasing number of practical use cases that can be
+       satisfied by a single signature most of the time but which still
+       require the use of scripts sometimes.  With just MAST---and not
+       taproot---those use cases would need to always use MAST.  MAST
+       could also be used for single-sig spends but it would require
+       larger transactions and more fees than a pure single-sig
+       construction, so single-sig users would probably not use MAST.
+       That would create a clear divide for chain analysis between
+       spends that use MAST and spends that don't.
+
+       Taproot eliminates that divide by allowing cheap single-sig
+       spends that are identical in appearance to those of users who can
+       use single-sig but who also have fallback scripts (though
+       actually spending using a fallback script will be identifiable onchain).
+       This creates a larger anonymity set than doing MAST and schnorr
+       separately as long as there really is a group of people who
+       sometimes spend using a single signature and other times spend
+       using a script.
+
+    2. {:#tap2} Anon asks, "Is taproot actually cheaper than bare MAST
+       and schnorr separately?"  Earlier in the email, Anon claimed that
+       taproot saves 67 bytes compared to MAST+schnorr for key-path
+       spending but adds 67 bytes for script-path spending.
+
+       Towns points out a redundant data field in Anon's calculation and
+       shows that taproot actually only adds about 33 bytes in the
+       script-path spending case, making the cost-benefit analysis
+       asymmetric in favor of taproot.  David Harding [notes][harding
+       tap] that the extra size (which translates to 8.25 vbytes) is
+       quite small compared to all the other data a script-path spender
+       would need to provide to spend a UTXO (e.g. 41 vbytes of input
+       data, 16-vbyte signatures or other witnesses of various sizes,
+       one or more 8-vbyte merkle nodes, and the script to execute).
+
+    3. {:#tap3} Anon asks, "Is taproot riskier than bare MAST and
+       schnorr separately given the new crypto?"
+
+       Towns replies that he "doesn't think so; most of the risk for
+       either of those is in getting the details right. [...] Most of
+       the complicated crypto parts are at the application layer:
+       [MuSig][topic musig], threshold signatures, adaptor signatures,
+       scriptless scripts, etc."  He also links several resources for
+       those wanting to learn more ([1][taplearn1], [2][taplearn2],
+       [3][taplearn3]).
+
+    4. {:#tap4} Anon asks, "couldn't we forego the [Nothing Up My
+       Sleeve]<!-- prevent link --> [NUMS][] point requirement and be able to check if it's a
```

Comment:
When viewing the newletter online, I'm seeing

Anon asks, “couldn’t we forego the [Nothing Up My Sleeve] NUMS point requirement...

is the formatting correct?

Reply:
"[Nothing Up My Sleeve]" is an editorial interpolation from @harding. The original is "couldn't we forego the NUMS point requirement" so the formatting is correct.

Reply:
> "[Nothing Up My Sleeve]" is an editorial interpolation from @harding. The original is "couldn't we forego the NUMS point requirement" so the formatting is correct.

Thanks, John!